docs/build/tutorials/smart-contract-development/paymasters/erc20fixed.md

```diff
@@ -312,9 +255,9 @@ describe.only("ERC20fixedPaymaster", function () {
     const finalPaymasterBalance = await provider.getBalance(paymasterAddress);
 
     expect(await greeter.greet()).to.equal("Hola, mundo!");
-    expect(initialPaymasterBalance).to.be.gt(finalPaymasterBalance);
+    expect(initialPaymasterBalance > finalPaymasterBalance).to.be.true;
```

Comment:
nit: we should be using `to.be.gt(...)` because if this `expect()` fails, the error message is much easier to understand what went wrong.

`Expected 100 to be greater than 101`
vs
`Expected false to be true`

Reply:
can't use `gt` (and similars) with Javascript BigInt 🫤